webpack.config.js

```diff
@@ -50,6 +54,12 @@ module.exports = {
         use: 'mocha-loader',
         exclude: /node_modules/,
       },
+      {
+        test: /node_modules[\/\\](iconv-lite)[\/\\].+/,
```

Comment:
What's this all about?

Reply:
This was to fix a bug that was happening due to electron having issues rendering an icon (this was the error with the '?' symbol).

Reply:
DOPE